src/main/java/com/obj/nc/flows/dataSources/PollFlowAdapter.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2021 the original author or authors.
+ * This file is part of Notiflow
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/.
+ */
+
+package com.obj.nc.flows.dataSources;
+
+import com.obj.nc.functions.processors.genericDataPersister.GenericDataPersister;
+import com.obj.nc.functions.processors.jsonNodeToGenericDataTransformer.Data2GenericDataTransformer;
+import com.obj.nc.functions.processors.jsonNodeToGenericDataTransformer.JsonNodeFilterAndTransformer;
+import com.obj.nc.repositories.GenericDataRepository;
+import com.obj.nc.utils.JsonUtils;
+import lombok.RequiredArgsConstructor;
+import org.springframework.integration.dsl.IntegrationFlowBuilder;
+import org.springframework.integration.dsl.StandardIntegrationFlow;
+import org.springframework.integration.dsl.Transformers;
+import org.springframework.integration.json.ObjectToJsonTransformer;
+import org.springframework.stereotype.Component;
+
+import static com.obj.nc.flows.dataSources.GenericDataConvertingFlowConfiguration.GENERIC_DATA_CONVERTING_FLOW_ID_INPUT_CHANNEL_ID;
+
+@Component
+@RequiredArgsConstructor
+public class PollFlowAdapter {
```

Comment:
Chosse more specific name. This class is more somthing lie GenericDataPersistAndTransformationFlow

Reply:
Renamed